Source/WebCore/testing/Internals.h

```diff
@@ -607,6 +607,7 @@ class Internals final
     ExceptionOr<String> pageSizeAndMarginsInPixels(int pageNumber, int width, int height, int marginTop, int marginRight, int marginBottom, int marginLeft) const;
 
     ExceptionOr<float> pageScaleFactor() const;
+    void setPageScaleFactor(float scaleFactor, DOMPromiseDeferred<void>&&);
```

Comment:
This was intentionally moved to testRunner and shouldn't be re-added here.

Reply:
It was, but I think there's value in having it.

We have quite a few places that originate a page scale change within the WebContent process, and then we take a slightly different code path to share it to all the other WebContent process.

From what I can see: WKBundlePageSetScaleAtOrigin, HistoryController, DrawingArea::scaleViewToFitDocumentIfNeeded and RemoteLayerTreeDrawingAreaMac::applyTransientZoomToPage are all examples of that.

As long as we have those, it seems valuable to also have a matching entry point for test code.

Maybe there's a way to get rid of all of those, and then we can avoid having the impl code as well as the test code?

`kWKLayoutModeDynamicSizeComputedFromMinimumDocumentSize` appears to be API, but maybe it's not used.